### ISSUE TITLE
Not being able to use HTMLKit via Xcode Package Manager

### ISSUE THREAD

**Issue by pendowski** [iabudiab/HTMLKit#41]
When I try to add HTMLKit as a dependency I get an error message at the Resolving package dependencies stage:

```
because HTMLKit >=0.9.4 contains incompatible tools version and root depends on HTMLKit 3.1.0..<4.0.0, version solving failed.
```

Tested on Xcode 11.4 and 12 beta.

**Comment by iabudiab** [comment]
@pendowski Hey there, I'll release an update some time later today 😉

**Comment by iabudiab** [comment]
@pendowski Just released v4.0.0. Swift tooling is updated to 5.1. Should word fine with Xcode 11.4. I haven't tested it with Xcode 12 beta yet.

I'll keep this open for while. Please let me know if it worked for you.

**Comment by iabudiab** [comment]
Works in the last version. Please reopen if it doesn't work for you.
